### PR TITLE
{Fabric Version} Add `keepEffects` Gamerule

### PR DESCRIPTION
**Pull request by FirstMegaGame4**

### CODE REVIEW

File: src/main/java/net/reikeb/notenoughgamerules/Gamerules.java [hunk 1]
```diff
@@ -41,5 +42,6 @@ public static void setupGamerules() {
         RAW_MEAT_HUNGER = GameRuleRegistry.register("rawMeatHunger", GameRules.Category.PLAYER, GameRuleFactory.createIntRule(30));
         SKY_HIGH = GameRuleRegistry.register("skyHigh", GameRules.Category.MISC, GameRuleFactory.createIntRule(-64));
         TNT_EXPLODES = GameRuleRegistry.register("tntExplodes", GameRules.Category.MISC, GameRuleFactory.createBooleanRule(true));
+        KEEP_EFFECTS = GameRuleRegistry.register("keepEffects", GameRules.Category.PLAYER, GameRuleFactory.createBooleanRule(false));
```
Comment (Max094Reikeb) [inline src/main/java/net/reikeb/notenoughgamerules/Gamerules.java]: Please keep creation and definition of Gamerules in alphabetical order

File: src/main/java/net/reikeb/notenoughgamerules/NotEnoughGamerules.java [hunk 2]
```diff
@@ -3,11 +3,13 @@
 import net.fabricmc.api.ModInitializer;
 import net.fabricmc.fabric.api.entity.event.v1.EntitySleepEvents;
 
+import net.fabricmc.fabric.api.entity.event.v1.ServerPlayerEvents;
```
Comment (Max094Reikeb) [inline src/main/java/net/reikeb/notenoughgamerules/NotEnoughGamerules.java:5]: No space before that import, but one space after it

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]

File: src/main/java/net/reikeb/notenoughgamerules/events/AfterRespawnListener.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package net.reikeb.notenoughgamerules.events;
+
+import net.fabricmc.fabric.api.entity.event.v1.ServerPlayerEvents;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.world.GameRules;
+import net.reikeb.notenoughgamerules.Gamerules;
```
Comment (Max094Reikeb) [inline src/main/java/net/reikeb/notenoughgamerules/events/AfterRespawnListener.java:9]: Sort the imports + spaces between line 3-4 and 6-7

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/not_enough_gamerules/lang/en_us.json [hunk 4]
```diff
@@ -24,5 +24,7 @@
   "gamerule.skyHigh": "SkyHigh",
   "gamerule.skyHigh.description": "Defines the minimal high the player has to be before taking damages",
   "gamerule.tntExplodes": "TNT explosions",
+  "gamerule.keepEffects": "Keep effects",
+  "gamerule.keepEffects.description": "Keeps player's effects after death",
```
Comment (Max094Reikeb) [inline src/main/resources/assets/not_enough_gamerules/lang/en_us.json]: Here too it must be sorted in alphabetical order :)

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/not_enough_gamerules/lang/en_us.json (hunk 4 above)
Comment (FirstMegaGame4) [reply src/main/resources/assets/not_enough_gamerules/lang/en_us.json]: :kappa:

File: src/main/java/net/reikeb/notenoughgamerules/Gamerules.java [hunk 5]
```diff
@@ -23,6 +23,7 @@
     public static GameRules.Key<GameRules.IntRule> RAW_MEAT_HUNGER;
     public static GameRules.Key<GameRules.IntRule> SKY_HIGH;
     public static GameRules.Key<GameRules.BooleanRule> TNT_EXPLODES;
+    public static GameRules.Key<GameRules.BooleanRule> KEEP_EFFECTS;
```
Comment (Max094Reikeb) [inline src/main/java/net/reikeb/notenoughgamerules/Gamerules.java]: Here. Alphabetical order please.

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]

File: src/main/java/net/reikeb/notenoughgamerules/Gamerules.java (hunk 5 above)
Comment (FirstMegaGame4) [reply src/main/java/net/reikeb/notenoughgamerules/Gamerules.java]: image

Comment (Max094Reikeb) [reply src/main/java/net/reikeb/notenoughgamerules/Gamerules.java]: Yes I know

**Review by Max094Reikeb — APPROVED** [review]

**Pull request merged**